ndcube/asdf/converters/tablecoord_converter.py

```diff
@@ -0,0 +1,85 @@
+from asdf.extension import Converter
+
+
+class TimeTableCoordConverter(Converter):
+    tags = ["tag:sunpy.org:ndcube/extra_coords/table_coord/timetablecoordinate-*"]
+    types = ["ndcube.extra_coords.table_coord.TimeTableCoordinate"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from ndcube.extra_coords.table_coord import TimeTableCoordinate
+
+        table = node.get("table")
+        names = node.get("names")
+        physical_types = node.get("physical_types")
+        reference_time = node.get("reference_time")
+        timetablecoordinate = TimeTableCoordinate(
+            table, names=names, physical_types=physical_types, reference_time=reference_time)
+
+        return timetablecoordinate
+
+    def to_yaml_tree(self, timetablecoordinate, tag, ctx):
+        node = {}
+        node["table"] = timetablecoordinate.table
+        node["names"] = timetablecoordinate.names
+        node["mesh"] = timetablecoordinate.mesh
+        if timetablecoordinate.physical_types is not None:
+            node["physical_types"] = timetablecoordinate.physical_types
+        node["reference_time"] = timetablecoordinate.reference_time
+
+        return node
+
+
+class QuantityTableCoordinateConverter(Converter):
+    tags = ["tag:sunpy.org:ndcube/extra_coords/table_coord/quantitytablecoordinate-*"]
+    types = ["ndcube.extra_coords.table_coord.QuantityTableCoordinate"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from ndcube.extra_coords.table_coord import QuantityTableCoordinate
+
+        unit = node.get("unit")
+        table = node.get("table")
```

Comment:
table and unit are required by the schema so the `get` can be removed (similar to the above comment for `TimeTableCoordinate`).

Reply:
Ok